### DISCUSSION TITLE
GIS Extension

### DISCUSSION THREAD

**Discussion by evonhoene** [JoelForamitti/agentpy#44] [Q&A]
I am planning on using AgentPy to build an ABM, but I need to be able to integrate the ABM with GIS. Is there anyway to add a GIS extension with AgentPy? If so, how would I do this, or are there any resources that may help?

**Comment by sunaina2821** [comment]
Although I am not sure how you'd go about doing it but this issue has been mentioned here and the short discussion might provide some pointers - [https://github.com/JoelForamitti/agentpy/discussions/11#discussioncomment-1535864](url)

**Reply by jofmi** [reply]
The link doesn't work for me, but yes, there's been some suggestions here: https://github.com/JoelForamitti/agentpy/discussions/11
